src/Pages/CreatorPage.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+// import propTypes from 'prop-types';
+import AuthorInfo from '../Components/creatorPage/AuthorInfo';
+import CreatedWorksCard from '../Components/creatorPage/OtherWorks';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+function CreatorPage() {
+  const [posts, setPosts] = useState();
```

Comment:
Can you rename these states to represent what information is being held here? That will make the code more legible!